views/_partials/stackshare-alert.pug

```diff
@@ -0,0 +1,5 @@
+.alert.alert-primary.alert-dismissible.fade.show(role='alert')
+  center 
```

Comment:
`center` is from the 90s :P Just use `text-center` in the alert and get rid of this.